tox.ini

```diff
@@ -61,6 +61,37 @@ commands =
     test: pytest --cov={env:PY_MODULE} --cov-report=term-missing:skip-covered --doctest-modules {posargs:src/{env:PY_MODULE}}
 
 
+##
+# Black code formatting
+##
+
+[testenv:black]
+
+description = run Black (linter)
+
+basepython = {[default]basepython}
+
+skip_install = True
+
+deps =
+    black==19.10b0
+
+setenv =
+    BLACK_LINT_ARGS=--check
+
+commands =
+    black {env:BLACK_LINT_ARGS:} setup.py src
+
+
+[testenv:black-reformat]
+
+description  = {[testenv:black]description} and reformat
+basepython   = {[testenv:black]basepython}
+skip_install = {[testenv:black]skip_install}
+deps         = {[testenv:black]deps}
+commands     = {[testenv:black]commands}
+
```

Comment:
So this works because the lint args are absent?

Reply:
Yeah, `{env:BLACK_LINT_ARGS:}` evaluates to `""` in the `black-reformat` environment.